### ISSUE TITLE
our server has four RTX2080 GPUs. Does this tutorial also fit into them? 

### ISSUE THREAD

**Issue by YOONSEOKHEO** [NVIDIA/DeepLearningExamples#110]
In my lab, we have two servers, each of which consists of two RTX 2080 GPUs respectively.
These tutorials are optimized for Tesla Tensor Cores as mentioned. 
Am I okay to understand that these codes are also well-executed under RTX2080 GPUs?

**Comment by nvcforster** [comment]
We typically perform our testing on V100 GPUs prior to release; however, it is expected to run on the RTX 2080 with good performance still. The RTX 2080 has Tensor Cores, although the performance of our current V100 flagship model found in DGX workstations and servers will be greater due to factors such as number of processing cores per chip. An important aspect is the available memory and batch size chosen. The RTX 2080 has 8GB of memory while the V100 models have up to 32GB, so the batch per GPU will need to be adjusted proportionally. In examples where gradient accumulation is implemented, this is usually the preferred option over reducing batch size since other hyperparameters (e.g., learning rate) can remain the same as in the documentation.

It would be useful for others to know if you find better hyperparameters for these examples in your configuration. If you don't mind sharing, please post your findings here.

**Comment by macanv** [comment]
How do I choose compute capability when I use RTX2080Ti?

**Comment by hwansoo** [comment]
> How do I choose compute capability when I use RTX2080Ti?

GeForce RTX 2080 Ti has compute capability of 7.5.
